tests/console/mariadb_srv.pm

```diff
@@ -33,8 +42,35 @@ sub run {
     }
     systemctl "status $mariadb", expect_false => 1, fail_message => 'mariadb should be disabled by default';
     systemctl "start $mariadb";
-    systemctl "status $mariadb";
-    assert_screen 'test-mysql_srv-1';
+    systemctl "is-active $mariadb";
+
+    # Test multiple instance configuration
+    # It is not supported in sle12sp2 and sle12sp3
+    if (!is_sle('<=12-SP3') && !is_ppc64le && !is_ppc64) {
+        assert_script_run "touch /etc/mynode1.cnf";
+        assert_script_run "touch /etc/mynode2.cnf";
```

Comment:
You could do all this same commands with [brace expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html) e.g. ` mkdir -p /var/lib/mysql/node{1,2}`

Reply:
I kept only the `systemctl` commands as same for visibility ;)